pantheon-wordpress-edge-integrations.php

```diff
@@ -11,6 +11,10 @@
 
 namespace Pantheon\EI\WP;
 
+define( 'PANTHEON_EDGE_INTEGRATIONS_VERSION', '0.1' );
```

Comment:
Can we automate this by pulling from the plugin header and then only update one place when we version bump?

Reply:
Great idea, see [e382774](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/pull/8/commits/e382774fd256fc781290bf2e2f7879c873d58477)